internal/common/labels.go

```diff
@@ -61,3 +65,11 @@ func addInstanceLabels(requestInstance *v1beta1.SSP, to map[string]string) {
 func copyLabel(from, to map[string]string, key string) {
 	to[key] = from[key]
 }
+
+func GetAppNameSelector(name string) labels.Selector {
+	appNameRequirement, err := labels.NewRequirement(AppKubernetesNameLabel, selection.Equals, []string{name})
+	if err != nil {
+		panic(fmt.Sprintf("Failed creating label selector for '%s=%s'", AppKubernetesNameLabel, name))
```

Comment:
I think this function should not panic on error, because the error can be caused by an incorrect value of `name` parameter.